packages/test-helpers/contracts/time/ClockMock.sol

```diff
@@ -5,55 +5,55 @@ import "@aragon/os/contracts/lib/math/SafeMath.sol";
 import "@aragon/os/contracts/lib/math/SafeMath64.sol";
```

Comment:
I've taken the liberty of renaming these contracts in #39.

- `ClockMock` --> `TimeHelpersMock`
- `TimeHelpersMock` --> `SharedTimeHelpersMock`

My main rationale for this is that I think `TimeHelpersMock` is the basic case, and is much more likely to be used as apps usually don't need a shared source of time.

When they do need that shared source of time, `SharedTimeHelpersMock` seems like an obvious name :).